python/cuml/cuml/linear_model/base_mg.pyx

```diff
@@ -38,7 +38,16 @@ from cuml.internals.array_sparse import SparseCumlArray
 class MGFitMixin(object):
 
     @cuml.internals.api_base_return_any_skipall
-    def fit(self, input_data, n_rows, n_cols, partsToSizes, rank, order='F', convert_index=np.int32):
+    def fit(
+        self,
+        input_data,
+        n_rows,
+        n_cols,
+        partsToSizes,
+        rank,
+        order='F',
+        convert_index=np.int32,
+    ):
```

Comment:
Just for my curiosity: do you know how it chooses between using
```python
    def fit_transform(
        self, X, y=None, sample_weight=None, *, convert_dtype=False
    ) -> CumlArray:
```
and this style of argument listing? I somehow expected it always to use the "one arg per line" style 🤷

Reply:
There's no autoformatting, you the writer have to pick. `ruff` and `black` will do

```
   def fit_transform(
        self, X, y=None, sample_weight=None, *, convert_dtype=False
    ) -> CumlArray:
```

if everything fits below the line limit _and_ you don't include a trailing comma (e.g. after `False` above). Otherwise it does the one arg per line. After years of seeing code that way, that's how I type it when I have to manual format too.